### DISCUSSION TITLE
Türschloss DLD

### DISCUSSION THREAD

**Discussion by T1gg1** [SukramJ/aiohomematic#1749] [General]
Hallo,

jedes Mal wenn Homeassistant neu gestartet wird, schließt das Schloss ab. Ist das so gewollt oder muss ich was konfigurieren?
VG

**Comment by SukramJ** [comment]
Du glaubst das man so was unglaublich dämliches bewusst in eine Integration einbaut?
Kontrollier mal die Auslöser deiner HA Automatisierungen bzw. CCU Programme.

**Reply by T1gg1** [reply]
Aufschließen wäre dämlich^^
Abschließen nicht.....
Ich habe keine Automatisierungen. Deswegen frage ich. Es kann ja auch ein Bug sein. Entschuldige, dass ich gefragt habe....

**Reply by SukramJ** [reply]
> Entschuldige, dass ich gefragt habe....

Kommt immer darauf an wie man fragt.

**Reply by T1gg1** [reply]
Es war eine normale Frage, ob es als Sicherheitsfeature gewollt ist🤷🏻‍♂️
Ich hatte keinen Angriff auf Euch vor.

**Reply by SukramJ** [reply]
> Es war eine normale Frage, ob es als Sicherheitsfeature gewollt ist

Es ist nicht die Aufgabe einer Integration vermeintliche Sicherheitsfeature ungefragt zu implementieren.

> Ich hatte keinen Angriff auf Euch vor.

Kein Problem.
